src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -137,13 +142,23 @@ class ConfigureIntegration extends AsyncView<Props, State> {
           </Form>
         )}
 
-        {integration.dynamicDisplayInformation?.configure_integration?.instructions.map(
-          (instruction, i) => (
-            <Alert type="info" key={i}>
-              <span dangerouslySetInnerHTML={{__html: singleLineRenderer(instruction)}} />
-            </Alert>
-          )
-        )}
+        <Alert type="info">
+          {instructions?.length === 1 ? (
+            <span
+              dangerouslySetInnerHTML={{__html: singleLineRenderer(instructions[0])}}
+            />
+          ) : (
+            <List symbol={<IconArrow size="xs" direction="right" />}>
```

Comment:
I think that we shouldn't render the List and the Alert components if `!instructions.length`. maybe we should verify it before